plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridIndexManager.java

```diff
@@ -54,9 +56,8 @@ protected void addObjectDeleteActions(
         @NotNull Map<String, Object> options
     ) {
         GenericTableIndex index = command.getObject();
-        actions.add(new SQLDatabasePersistAction(
-            "Drop Index",
-            "DROP INDEX " + index.getName() + " ON " + ((CubridTable) index.getTable()).getUniqueName()
-        ));
+        actions.add(new SQLDatabasePersistAction("Drop Index",
+        "DROP INDEX " + DBUtils.getQuotedIdentifier(index.getDataSource(), index.getName()) + " ON "
+        + ((CubridTable) index.getTable()).getFullyQualifiedName(DBPEvaluationContext.DDL)));

```

Comment:
Redundant cast to `CubridTable`
```suggestion
        "DROP INDEX " + DBUtils.getQuotedIdentifier(index.getDataSource(), index.getName()) + " ON "
        + index.getTable().getFullyQualifiedName(DBPEvaluationContext.DDL)));

```